app/Console/Commands/RebuildImages.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Poniverse\Ponyfm\Console\Commands;
+
+use Illuminate\Console\Command;
+use Poniverse\Ponyfm\Models\Image;
+use Symfony\Component\HttpFoundation\File\File;
+
+class RebuildImages extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rebuild:images';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Resizes all images to fit the specifications in Models/Image';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $images = Image::all();
+        foreach ($images as $image) {
```

Comment:
This is likely to run out of memory in production, where we have a rather large number of uploaded images that grows further over time. Implement this routine with [model chunking](https://laravel.com/docs/5.3/eloquent#chunking-results) to put an upper bound on its memory footprint.